integrations/cypress/README.md

```diff
@@ -93,3 +93,11 @@ after(async () => {
   cy.logout();
 });
 ```
+
+You may also log out _before_ each test to allow one to inspect the state after a test run.
+
+```js
+beforeEach(async () => {
+  cy.logout();
```

Comment:
Does this work?  I think this might throw an exception as it currently stands

Reply:
Hmm, good point, I don't think it will. Let me throw in a test to confirm. Speaking of which, I don't think we are running the tests for this in CI. I can add it to the workflow.

Reply:
I think calling `cy.logout()` should be idempotent or not blow up if you are not logged in.

We should possibly leave the README and fix `cy.logout()`

We cold pair on this later if you have time.

Reply:
I can take a stab first. We can pair tomorrow (my) morning if I am stuck if that sounds good with you.

Reply:
go for it man!  All yours